File: src/main/java/org/virajshah/monopoly/simulator/SimpleSimulation.java
package org.virajshah.monopoly.simulator;

import org.virajshah.monopoly.core.MonopolyGame;
import org.virajshah.monopoly.core.Player;
import org.virajshah.monopoly.logger.Logger;

public class SimpleSimulation {
    public static void main(String[] args) {
        Logger.init();

        MonopolyGame game = new MonopolyGame();

        game.addPlayer(new Player("Player 1"));
        game.addPlayer(new Player("Player 2"));
        game.addPlayer(new Player("Player 3"));
        game.addPlayer(new Player("Player 4"));

        for (int i = 0; i < 1000; i++) {
            game.runNextTurn();

            if (game.getPlayers().isEmpty() || game.getPlayers().size() == 1)
                System.exit(0);
        }

        Logger.save();
    }
}

File: src/main/java/org/virajshah/monopoly/simulator/SimpleSimulation.kt
package org.virajshah.monopoly.simulator

import org.virajshah.monopoly.core.MonopolyGame
import org.virajshah.monopoly.core.Player

object SimpleSimulation {
    @JvmStatic
    fun main(args: Array<String>) {
        val game = MonopolyGame()
        game.addPlayer(Player("Player 1"))
        game.addPlayer(Player("Player 2"))
        game.addPlayer(Player("Player 3"))
        game.addPlayer(Player("Player 4"))
        for (i in 0..999) {
            game.runNextTurn()
            if (game.players.isEmpty() || game.players.size == 1) {
                System.exit(0)
            }
        }
    }
}